src/Core/Framework/App/Lifecycle/AppLifecycle.php

```diff
@@ -239,7 +241,12 @@ private function updateApp(
 
         $this->updateCustomEntities($app, $manifest);
 
-        $this->permissionPersister->updatePrivileges($manifest->getPermissions(), $id);
+        $this->permissionPersister->updatePrivileges(
+            $manifest->getPermissions(),
+            $id,
+            $manifest->validatesPermissions() === false && $parameters->acceptPermissions,
```

Comment:
`validatesPermissions()` should be immediately deprecated for 6.8 and the condition here removed with that

Reply:
Not true. We will deprecate for 6.9, but in 6.8 we will change the default value from false to true.